lib/managers/files.js

```diff
@@ -575,6 +575,27 @@ Files.prototype.deletePermanently = function(fileID, callback) {
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+/*
+ * Retrieves a file that has been moved to the trash.
+ *
+ * API Endpoint: '/files/:fileID/trash'
+ * Method: GET
+ *
+ * @param {string} fileID - The ID of the file being requested
+ * @param {?Object} qs - Additional options can be passed with the request via querystring. Can be left null in most cases.
+ * @param {Function} callback - Called with nothing if successful, error otherwise
```

Comment:
This comment needs to be updated; the callback is called with information about the file when successful